### PR TITLE
Windows: resolved merge conflicts and added tests

### PR DESCRIPTION
**Pull request by gdavison**
This adds support for installing on Windows using the `install_method` of `windows`. It uses the Chocolatey package manager.

I have made the Windows tests conditional on setting the `TEST_WINDOWS` environment variable to a truthy value. I tried using vagrant boxes from [https://github.com/joefitzgerald/packer-windows] and [https://github.com/boxcutter/windows],  but I was not able to make them work. The box used is approximately 3 GB to download. Provisioning only seems to work if you login at boot time; annoying, but it works.

### CODE REVIEW

**Comment by gdavison** [comment]
The box I was using appears to have been taken offline today. I've changed to a new box from the same source. You no longer need to log in, but kitchen seems to try on first run to use ssh to connect. I cancel the first run with ^C, then re-run the test, and it uses winrm. No idea why.

**Comment by johnbellone** [comment]
What's the best way to test this?

**Comment by gdavison** [comment]
I've fixed some of the Windows VM stuff by manually setting some values that I thought Kitchen was supposed to set. I'm verifying that all tests pass right now, and will re-push when that's done.

**Comment by gdavison** [comment]
It's all updated. You can run `TEST_WINDOWS=1 kitchen test "^[^rs].*(ub|win)"`. It skips the `centos`, `runit`, and `source` tests, and runs the `windows` test.

**Comment by johnbellone** [comment]
This is failing for me. Are you using ChefDK?

**Comment by gdavison** [comment]
Yes, ChefDK 0.6.2 on Mac OS 10.10. What errors are you getting?

File: .kitchen.yml [hunk 1]
```diff
@@ -10,9 +10,22 @@ platforms:
   - name: ubuntu-12.04
   - name: centos-7.0
   - name: centos-6.5
+  <% if ENV['TEST_WINDOWS'] %>
```
Comment (johnbellone) [inline .kitchen.yml]: I do not see a reason why we need to keep the Windows test support on an environment variable? I've removed it locally and all looks good to me. Please remove this bit.

File: .kitchen.yml [hunk 2]
```diff
@@ -63,10 +85,23 @@ suites:
...
     run_list:
       - recipe[consul::default]
     attributes:
       consul:
         atlas_autojoin: true
         atlas_cluster: <%= ENV.fetch('ATLAS_CLUSTER', 'example/cluster') %>
-        atlas_token: <%= ENV.fetch('ATLAS_TOKEN', 'NOT_REAL') %>
\ No newline at end of file
+        atlas_token: <%= ENV.fetch('ATLAS_TOKEN', 'NOT_REAL') %>
+  <% if ENV['TEST_WINDOWS'] %>
```
Comment (johnbellone) [inline .kitchen.yml]: The same goes here as well.

**Pull request merged**